generation/anomaly_detection/2d_classifierfree_guidance_anomalydetection_tutorial.ipynb

```diff
@@ -0,0 +1,1033 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #22.    os.environ["CUDA_VISIBLE_DEVICES"] = "0"
</pre><p>I wouldn't include this here, it avoids any confusion the user may have with their environment.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1818/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>